chromium_src/BUILD.gn

```diff
@@ -156,9 +156,13 @@ static_library("chrome") {
       "//chrome/browser/ui/frame/window_frame_util.h",
       "//chrome/browser/win/chrome_process_finder.cc",
       "//chrome/browser/win/chrome_process_finder.h",
+      "//chrome/browser/win/chrome_select_file_dialog_factory.cc",
+      "//chrome/browser/win/chrome_select_file_dialog_factory.h",
       "//chrome/browser/win/titlebar_config.cc",
       "//chrome/browser/win/titlebar_config.h",
       "//chrome/browser/win/titlebar_config.h",
+      "//chrome/browser/win/util_win_service.cc",
```

Comment:
We also need to build the [mojom target](https://source.chromium.org/chromium/chromium/src/+/main:chrome/services/util_win/public/mojom/BUILD.gn;l=7) as a dep in https://github.com/electron/electron/blob/cf5fb4505ebde6a8347b45ec7e66a58d851944df/chromium_src/BUILD.gn#L185,

Reply:
@deepak1556 it's a public dep here: https://source.chromium.org/chromium/chromium/src/+/main:chrome/services/util_win/BUILD.gn;l=31

and i don't see Chromium include this separately anywhere 🤔  is there something i'm missing?